packages/@aws-cdk/aws-appmesh/test/test.virtual-node.ts

```diff
@@ -357,7 +360,147 @@ export = {
         test.done();
       },
     },
+
+    'when a grpc listener is added with a TLS certificate from ACM': {
+      'the listener should include the TLS configuration'(test: Test) {
+        // GIVEN
+        const stack = new cdk.Stack();
+
+        // WHEN
+        const mesh = new appmesh.Mesh(stack, 'mesh', {
+          meshName: 'test-mesh',
+        });
+
+        const cert = new acm.Certificate(stack, 'cert', {
+          domainName: '',
+        });
+
+        new appmesh.VirtualNode(stack, 'test-node', {
+          mesh,
+          listeners: [appmesh.VirtualNodeListener.grpc({
+            port: 80,
+            tlsCertificate: appmesh.TlsCertificate.acm({
+              acmCertificate: cert,
+              tlsMode: appmesh.TlsMode.STRICT,
+            }),
+          },
+          )],
+        });
+
+        // THEN
+
+        expect(stack).to(haveResourceLike('AWS::AppMesh::VirtualNode', {
+          Spec: {
+            Listeners: [
+              {
+                TLS: {
+                  Mode: appmesh.TlsMode.STRICT,
+                  Certificate: {
+                    ACM: {
+                      CertificateArn: {
+                        Ref: 'cert56CA94EB',
+                      },
+                    },
+                  },
+                },
+              },
+            ],
+          },
+        }));
+        test.done();
+      },
+    },
+
+    'when an http listener is added with a TLS certificate from file': {
+      'the listener should include the TLS configuration'(test: Test) {
+        // GIVEN
+        const stack = new cdk.Stack();
+
+        // WHEN
+        const mesh = new appmesh.Mesh(stack, 'mesh', {
+          meshName: 'test-mesh',
+        });
+
+        new appmesh.VirtualNode(stack, 'test-node', {
+          mesh,
+          listeners: [appmesh.VirtualNodeListener.http({
+            port: 80,
+            tlsCertificate: appmesh.TlsCertificate.file({
+              certificateChain: 'path/to/certChain',
+              privateKey: 'path/to/privateKey',
+              tlsMode: appmesh.TlsMode.STRICT,
+            }),
+          })],
+        });
+
+        // THEN
+        expect(stack).to(haveResourceLike('AWS::AppMesh::VirtualNode', {
+          Spec: {
+            Listeners: [
+              {
+                TLS: {
+                  Mode: appmesh.TlsMode.STRICT,
+                  Certificate: {
+                    File: {
+                      CertificateChain: 'path/to/certChain',
+                      PrivateKey: 'path/to/privateKey',
+                    },
+                  },
+                },
+              },
+            ],
+          },
+        }));
+        test.done();
```

Comment:
I like to add an empty line before the last `test.done()`, reads a little nicer IMO.